AKS-Hybrid/aks-edge-howto-connect-to-arc.md

```diff
@@ -42,10 +42,17 @@ Provide details of your Azure subscription in the [**aksedge-config.json**](http
 |`ClusterName` | string | Provide a name for your cluster. The default value is `hostname_cluster`. |
 |`Location` | string | The location of your resource group. Choose the location closest to your deployment. |
 |`SubscriptionId` | GUID | Your subscription ID. In the Azure portal, select the subscription you're using and copy/paste the subscription ID string into the JSON. |
+<<<<<<< Updated upstream
```

Comment:
@mgodfre3 Is this supposed to be a comment? The PR can't be merged with these markers visible.